### DISCUSSION TITLE
Practica finalizada de pastelería

### DISCUSSION THREAD

**Discussion by enriqueorea** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#223] [Show and tell]
Hola compañeros de launch X me gustaría compartirles la landing page de la pastelería ficticia que realice
https://enriqueorea.github.io/CakyStore/ 

OJO: únicamente es la maqueta en HTML/CSS/JS (algunas librerías de js). No cuenta con funcionalidad aún, posiblemente continué con el proyecto utilizando REACT para el FROnt y algún CMS para el BackEnd.
Se aceptan comentarios

Un saludo

**Comment by hackit92** [comment]
Me gustó mucho tu página, a modo de comentarios veo un par de detalles comenzando por el dashboard con pedidos e inventario para el pastelero y por otro lado el formulario esta un poco complicado para cotizar un pastel, me da la impresión de que la información solicitada no es adecuada para levantar un pedido. 
pd. Me gustó mucho el carrusel de los productos =D

**Reply by enriqueorea** [reply]
Te agradezco el comentario, trabajare para mejorar el formulario

**Comment by DanielGalindoRios** [comment]
Hola.

Deberias compartir el codigo, asi podemos ver como hiciste algunas caracteristicas y utilizaste algunas funciones.

Te quedo muy bien :)

Saludos

**Reply by enriqueorea** [reply]
con mucho gusto https://github.com/enriqueorea/CakyStore

**Comment by JuanSalvador73** [comment]
Excelente website, felicidades :D
